metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsPackageEntry.groovy

```diff
@@ -2,11 +2,16 @@ package life.qbic.metastats.datamodel
 
 class MetaStatsPackageEntry {
 
-    final String entryId
+    final String preparationSampleId
```

Comment:
@wow-such-code what was the differentiation between a sample ID and a sample code in our system again?

Reply:
I think there is no ID, it is a name I came up with. As far as I know there is sample type (e.g. Q_TEST_SAMPLE) and sample code (QXXXXES12)

Reply:
This identifier does not need to be a sample code, I intended it as something to use to be able to differentiate between samples fast

Reply:
I think to remember a discussion in the sample-tracking. I kind of recall that there was also a sample id